packages/react-core/src/demos/DescriptionList/examples/DescriptionListDrawer.tsx

```diff
@@ -0,0 +1,163 @@
+import React from 'react';
+import {
+  Gallery,
+  PageSection,
+  GalleryItem,
+  Card,
+  CardBody,
+  Drawer,
+  DrawerPanelContent,
+  DrawerContent,
+  DrawerContentBody,
+  DrawerHead,
+  DrawerActions,
+  DrawerCloseButton,
+  PageSectionVariants,
+  TextContent,
+  Text,
+  DrawerPanelBody,
+  DescriptionList,
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+  Button,
+  Tabs,
+  Tab
+} from '@patternfly/react-core';
+import { DashboardWrapper } from '@patternfly/react-core/dist/js/demos/DashboardWrapper';
+import PlusCircleIcon from '@patternfly/react-icons/dist/esm/icons/plus-circle-icon';
+
+export const DescriptionListDrawer: React.FunctionComponent = () => {
+  const drawerRef = React.useRef<HTMLDivElement>();
+  const [isExpanded, setIsExpanded] = React.useState(true);
+
+  const onExpand = () => {
+    drawerRef.current && drawerRef.current.focus();
+  };
+
+  const onCloseClick = () => {
+    setIsExpanded(false);
+  };
+
+  const onClickCard = () => {
+    setIsExpanded(true);
+  };
+
+  const panelContent = (
+    <DrawerPanelContent>
+      <DrawerHead>
+        <Text component="h1">ajay-test</Text>
+        <DrawerActions>
+          <DrawerCloseButton onClick={onCloseClick} />
+        </DrawerActions>
+      </DrawerHead>
+      <DrawerPanelBody>
+        <Tabs activeKey={0} variant={'default'} aria-label="Tabs in the box light variation example" role="region">
+          <Tab eventKey={0} title="Overview">
```

Comment:
@mcoker should the Tab components be in their own separate DrawerBody, or should Core be updated to match this React demo?

Actually curious as to whether we should even be using the Tabs component in these demos? Since the focus should be a DescriptionList in a Drawer, so using Tabs seems kind of random.

Reply:
+1 to tabs being random - I don't see why it's needed, and we have a similar [tables and tabs demo](https://www.patternfly.org/components/tabs/react-demos/tables-and-tabs/) I think folks could reference for this layout. Looks like tabs have since been removed from the PR, too. @andrew-ronaldson WDYT? If we didn't include tabs in this demo, we would create a follow-up for core to remove tabs from its [data list in a drawer demo](https://www.patternfly.org/components/description-list/html-demos/in-drawer/)

All that said, I think you would want separate `<DrawerPaneBody>`s since the one with tabs should not have padding, and the one with tabcontent should.